ee/modules/140-user-authz/hooks/internal/cluster_auth_rule.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package internal
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ClusterAuthorizationRule struct {
```

Comment:
Why don't you put this specs to `/modules/140-user-authz/hooks/internal`?

Reply:
removed